src/lib/components/ui/menu/menu-title.svelte

```diff
@@ -84,8 +84,12 @@
 			display: flex;
 			flex-direction: row;
 			align-items: center;
-			justify-content: space-between;
-			gap: 0.5rem;
+			justify-content: stretch;
+		}
+		.title {
+			display: flex;
```

Comment:
I don't think you need to have `display: flex;` here. This is only needed on the parent.
```suggestion
```